code/modules/jobs/job_types/prisoner.dm

```diff
@@ -0,0 +1,25 @@
+/datum/job/prisoner
+	title = "Prisoner"
+	flag = PRISONER
+	department_head = list("Head of security")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = "the head of security and warden"
+
+	outfit = /datum/outfit/job/prisoner
+
+	display_order = JOB_DISPLAY_ORDER_PRISONER
+
+/datum/outfit/job/prisoner
+	name = "prisoner"
+	jobtype = /datum/job/prisoner
+
+	uniform = /obj/item/clothing/under/rank/prisoner
+	shoes = /obj/item/clothing/shoes/sneakers/orange
+	id = /obj/item/card/id/prisoner
+	ears = 0
```

Comment:
shouldnt these be null

Reply:
They probably should be. Thanks for taking the time to review.